tests/e2e/Services/Teams/TeamsBaseServer.php

```diff
@@ -281,4 +282,101 @@ public function testDeleteUserUpdatesTeamMembershipCount($data)
         $this->assertIsInt($response['body']['total']);
         $this->assertEquals(true, $dateValidator->isValid($response['body']['$createdAt']));
     }
+
+    public function testTeamDeleteUpdatesUserMembership()
+    {
+        $new_users = [];
+
+        /**
+         * Create 5 new users and add their IDs to an array
+         */
+        for ($i = 0; $i < 5; $i++) {
+            $user = $this->client->call(Client::METHOD_POST, '/users', array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ], $this->getHeaders()), [
+                'userId' => ID::unique(),
+                'email' => 'newuser' . $i . '@localhost.test',
+                'password' => 'password',
+                'name' => 'New User ' . $i,
+            ], false);
+
+            $user_body = json_decode($user['body'], true);
+
+            $this->assertEquals(201, $user['headers']['status-code']);
+            $this->assertEquals('newuser' . $i . '@localhost.test', $user_body['email']);
+            $this->assertEquals('New User ' . $i, $user_body['name']);
+            $this->assertEquals($user_body['status'], true);
+
+            array_push($new_users, $user_body['$id']);
```

Comment:
PHP's shorthand notation for this is:

```suggestion
            $new_users[] = $user_body['$id'];
```

Reply:
Did not know that, thanks a lot! I'll make this change 💡